src/dotnet-interactive/Connection/StdIoKernelConnector.cs

```diff
@@ -99,7 +100,9 @@ public async Task<ProxyKernel> CreateRootProxyKernelAsync()
                     {
                         ["DOTNET_INTERACTIVE_SKIP_FIRST_TIME_EXPERIENCE"]  = "1",
                         ["DOTNET_SKIP_FIRST_TIME_EXPERIENCE"]  = "1",
-                        ["DOTNET_DbgEnableMiniDump"] = "0" // https://docs.microsoft.com/en-us/dotnet/core/diagnostics/dumps
+                        ["DOTNET_DbgEnableMiniDump"] = "0", // https://docs.microsoft.com/en-us/dotnet/core/diagnostics/dumps
+                        ["DOTNET_CLI_UI_LANGUAGE"] = GetCurrentUICulture(),
+                        ["DOTNET_CLI_CULTURE"] = GetCurrentCulture()
```

Comment:
A similar change will also need to be made in the Polyglot Notebooks extension when it launches the `dotnet-interactive` process, here: 

https://github.com/dotnet/interactive/blob/main/src/polyglot-notebooks-vscode-common/src/extension.ts#L130-L157

Reply:
done